README.md

```diff
@@ -65,3 +65,29 @@ You can set the file extension used by the UI to download a result from your fun
 * `environment`
 
 Set timeouts etc
+
+# OpenFaaS Template Store
+
+The Template Store are all the templates from OpenFaaS and OpenFaaS-Incubator organizations, along with user-defined templates which were approved by the community.
+
+## How to submit to the Template Store
+
```

Comment:
Suggest we reiterate the contribution statement somewhere within this section:

`If you'd like to make a submission then raise an issue to propose it. This should follow the [CONTRIBUTION guide](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) for OpenFaaS.`

I think the first sentence is covered by point 1, so its a matter of making the contributing guide link available.

Reply:
Done by transforming point one as you proposed ✔️